openedx/core/djangoapps/user_authn/api/helper.py

```diff
@@ -37,6 +38,7 @@ class RegistrationFieldsContext(APIView):
         'specialty',
         'marketing_emails_opt_in',
     ]
+    userProfile_fields = [field.name for field in UserProfile._meta.get_fields()]
```

Comment:
python uses snake case:
```suggestion
    use_profile_fields = [field.name for field in UserProfile._meta.get_fields()]
```